Pcap++/header/WinPcapLiveDevice.h

```diff
@@ -32,17 +32,17 @@ namespace pcpp
 		WinPcapLiveDevice& operator=(const WinPcapLiveDevice& other);
 
 	public:
-		virtual LiveDeviceType getDeviceType() const
+		virtual LiveDeviceType getDeviceType() const override
```

Comment:
The `virtual` can be removed if an `override` keyword is present.

Reply:
Yes, `virtual` could be omitted here.